config.env.py

```diff
@@ -20,8 +19,17 @@
 
 # Google OpenID Connect SSO config
 GOOGLE_ISSUER = env.get('GOOGLE_ISSUER', 'https://accounts.google.com')
-GOOGLE_CLIENT_ID = env.get('GOOGLE_CLIENT_ID', '')
-GOOGLE_CLIENT_SECRET = env.get('GOOGLE_CLIENT_SECRET', '------')
+GOOGLE_CLIENT_ID = env.get('GOOGLE_CLIENT_ID', '---')
+GOOGLE_CLIENT_SECRET = env.get('GOOGLE_CLIENT_SECRET', '---')
```

Comment:
Why not just leave these blank?

Reply:
Good question. I don't remember now but Id have to guess it might've thrown some error if I added it to GOOGLE_CLIENT_ID